src/components/Welcome/Welcome.jsx

```diff
@@ -1,6 +1,12 @@
-// import the React library here
-// import a css file containig article styles
+import React from 'react';
+import './Welcome.css';
 
-// Create a Welcome function returning the HTML of welcome block
+function Welcome() {
```

Comment:
```suggestion
export default function Welcome() {
```
You can also export the functions like that